src/gshhg_version.h

```diff
@@ -52,7 +52,6 @@ struct GSHHG_VERSION {
 };
 
 /* Prototypes */
-EXTERN_MSC int gshhg_get_version (const char* filename, struct GSHHG_VERSION *gshhg_version);
```

Comment:
What if someone whats to know the GSHHG version?

Reply:
They can, this is a sub-function of the gshhg_version function.